src/test/java/uk/gov/ons/ssdc/caseprocessor/messaging/RefusalReceiverTest.java

```diff
@@ -85,4 +85,59 @@ public void testRefusal() {
         .logCaseEvent(
             eq(caze), eq("Refusal Received"), eq(EventType.REFUSAL), eq(event), eq(message));
   }
+
+  @Test
```

Comment:
I think there should also be a test case which doesn't set the `eraseData` flag to prove it defaults to not erasing.